src/client/app/api/queries/saksbehandlerQueries.ts

```diff
@@ -71,6 +72,20 @@ export const useReserverOppgaveMutation = (options: UseMutationOptions<OppgaveSt
 		...options,
 	});
 };
+
+export const useEndreReservasjoner = (options: UseMutationOptions<OppgaveStatus, Error, EndreOppgaveType[]> = {}) => {
+	const queryClient = useQueryClient();
+	return useMutation({
+		...options,
+		mutationFn: (data) => axiosInstance.post(apiPaths.endreReservasjoner, data),
+		onSuccess: () => {
+			Promise.all([
```

Comment:
Berre flisespikkeri, ikkje noko feil altså, men når ein ikkje bruker oppretta Promise til noko, så blir vel igrunn dette det samme som å skrive
`
queryClient.refetchQueries(apiPaths.saksbehandlerReservasjoner);
queryClient.refetchQueries(apiPaths.avdelinglederReservasjoner);
`
uten Promise.all rundt. Begge to køyre i parallell uten å vente på noko?

Reply:
japp, det er sant. Skulle vært sånn her hvis det skulle vært noe vits

				Promise.all([
					queryClient.invalidateQueries(apiPaths.hentOppgavekoer),
					queryClient.invalidateQueries(apiPaths.hentOppgaveko(id)),
				]).then(() => {
					if (callback) callback();
				});